### ISSUE TITLE
Same entity?

### ISSUE THREAD

**Issue by nataled** [geneontology/pathways2GO#227]
So far as I can tell, R-HSA-6793516 "K48-linked polyubiquitin chain [cytosol]" and R-HSA-912740 "K48polyUb [cytosol]" are the same entity. The only difference is the name. Both are type 'Polymer' and both have R-HSA-912722 "K48-Ub [cytosol] " as the participating repeated unit. 

Not an issue for me (I'll just map them both to the same PRO term), but in case this is unintended and simplification is desired.

**Comment by deustp01** [comment]
Yes, these two instances are duplicates. Fixed by switching all referrers of R-HSA-6793516 to refer to R-HSA-912740 instead and deleting R-HSA-6793516 from Reactome. The change will be visible on the public website and database with our March 2025 release.
